src/program.js

```diff
@@ -67,10 +67,14 @@ export class Program {
         if (cmd) {
           prefix = `${cmd}: `;
         }
-        console.error(prefix + error.toString());
+
+        console.error();
```

Comment:
ah, yes, good idea. Hopefully this will all look nicer once [logging is in place](https://github.com/mozilla/web-ext/issues/33).